doc/index.html

```diff
@@ -745,7 +751,7 @@ <h3 id="command-line-options">Command-line options</h3>
  Unless --forbid_null is true, a <code>null</code> value will still be used
  if no other value can be passed as an argument even if --null-ratio=0.
 
- Randoop never uses <code>null</code> for receiver values. [default 0.0]</li>
+ Randoop never uses <code>null</code> for receiver values. [default 0.05]</li>
```

Comment:
This manual change is incorrect; it wasn't reverted when the variable default was reverted.